index.html

```diff
@@ -309,21 +309,25 @@ <h3>
 
           <ol>
             <li>
+Remove the <code>did:web:</code> prefix.
+            </li>
+            <li>
 Replace ":" with "/" in the method specific identifier to obtain the fully
 qualified domain name and optional path.
             </li>
             <li>
-If the domain contains a port percent decode the colon.
+If the domain contains a port, percent decode the colon.
             </li>
             <li>
 Generate an HTTPS URL to the expected location of the DID document by
 prepending <code>https://</code>.
             </li>
             <li>
-If no path has been specified in the URL, append <code>/.well-known</code>.
+If no path has been specified in the resulting URL, or if the path does not end
+in a <code>/</code> character, append <code>/</code>.
             </li>
             <li>
-Append <code>/did.json</code> to complete the URL.
+Append <code>did.json</code> to complete the URL.
```

Comment:
Oh, hrm... this might be ok... really, we should just be telling people to do the whole file path. Why aren't we doing that?

Reply:
I don't think we have rough consensus there -- this is what I mean, specifically: https://github.com/w3c-ccg/did-method-web/issues/52

Reply:
let''s leave this to a separate issue/PR. I think we're going to need more time to resolve issue #52 faster than we can find consensus for this PR.